docs/monitoring.md

```diff
@@ -189,10 +189,26 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>A list of all applications</td>
   </tr>
   <tr>
+     <td></td>
+     <td><code>?status=[completed|running]</code> list only applications in the state</td>
+  </tr>
+  <tr>
+     <td></td>
+     <td><code>?minDate=[date]</code> earliest date to list. e.g <code>?minDate=2015-02-10</code></td>
+  </tr>
+  <tr>
+     <td></td>
+     <td><code>?maxDate=[date]</code> latest date to list. e.g <code>?maxDate=2015-02-03T16:42:40.000GMT</code></td>
```

Comment:
maybe mention both date formats under `minDate`, and then just say this can take the same formats as `minDate`?  It should be pretty obvious that both versions work for either option, but just to be a little more clear.